ui/src/components/Wallet.jsx

```diff
@@ -33,17 +33,31 @@ export default function Wallet() {
 
       <List>
         {Array.isArray(purses) && purses.length > 0 ? (
-          purses.map(({ pursePetname, brandPetname, brandBoardId, amount: { value }}) => (
-            <ListItem key={pursePetname} value={pursePetname} divider>
-              <ListItemIcon className={classes.icon}>
-                <PurseIcon />
-              </ListItemIcon>
-              <ListItemText
-                primary={pursePetname}
-                secondary={<><b>{value} {brandPetname}</b> <i>({brandBoardId})</i></>}
-              />
-            </ListItem>
-          ))
+          purses.map(
+            ({
+              pursePetname,
+              brandPetname,
+              brandBoardId,
+              amount: { value },
+            }) => (
+              <ListItem key={pursePetname} value={pursePetname} divider>
+                <ListItemIcon className={classes.icon}>
+                  <PurseIcon />
+                </ListItemIcon>
+                <ListItemText
+                  primary={pursePetname}
+                  secondary={
+                    <>
```

Comment:
Not a blocker, just my curiousity. What does an empty element do in jsx?

Reply:
It's creating a JSX element. In this case it looks like we wanted to have a parent element for the siblings <b> and <i>